azavea/raster-foundry
# Add visibility field to Uploads table and model

## Overview

This PR adds a `visibility` field to the `Upload` model and database table, updates the swagger spec, and adds the necessary migration.

### Checklist

~~- [ ] Styleguide updated, if necessary~~

### Demo

![shot-2017-04-04_11-55-51](https://cloud.githubusercontent.com/assets/18290666/24666220/b571f982-192d-11e7-83e9-cb82a870f669.jpg)



## Testing Instructions

* apply migrations
* spin up the server and using Postman or the like, make a `POST` request to `http://localhost:9091/api/uploads/` with a payload similar to this -->
```
{
  "organizationId": "dfac6307-b5ef-43f7-beda-b9f208bb7726",
  "uploadStatus": "UPLOADING",
  "fileType": "GEOTIFF",
  "uploadType": "LOCAL",
  "files": [],
  "datasource": "697a0b91-b7a8-446e-842c-97cda155554d",
  "metadata": {},
  "visibility":"PRIVATE"
}
```
* check that the swagger spec is up-to-date and correct


Connects #1368
